WcaOnRails/app/models/competition.rb

```diff
@@ -66,10 +65,10 @@ class Competition < ActiveRecord::Base
     self.persisted? || (name.length <= MAX_NAME_LENGTH && name =~ VALID_NAME_RE)
   end
   validates :name, length: { maximum: MAX_NAME_LENGTH },
-                   format: { with: VALID_NAME_RE, message: INVALID_NAME_MESSAGE }
+                   format: { with: VALID_NAME_RE, message: I18n.t('competitions.errors.invalid_name_message') }
```

Comment:
Hmm... doesn't this call to `I18n.t` run exactly once on server startup? Maybe this can be converted into a block somehow?